plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/catalog/snowflake/TestIcebergSnowflakeCatalogConnectorSmokeTest.java

```diff
@@ -610,6 +610,14 @@ public void testExecuteDelete()
                 .hasMessageMatching("Failed to close manifest writer");
     }
 
+    @Test
+    @Override
+    public void testTruncateTable()
+    {
+        assertThatThrownBy(super::testTruncateTable)
+                .hasMessageContaining("Failed to close manifest writer");
```

Comment:
is this expected failure, a bug, or a missing feature?

Reply:
This is an expected failure because Snowflake catalog doesn't support write operations. We could improve the error message though.

Reply:
yeah , "Failed to close manifest writer" isn't exactly same as "doesn't support write operations"